eureka_ml_insights/data_utils/aime_utils.py

```diff
@@ -0,0 +1,68 @@
+# Part of this file was authored by authors of the Kitab dataset (https://huggingface.co/datasets/microsoft/kitab)
+# Some code in this file is copied from the original source repository and then adapted to fit this repository.
+# The original license for this code is Community Data License Agreement - Permissive - Version 2.0
+
+import ast
+import re
+import urllib.request
+from dataclasses import dataclass
+
+import pandas as pd
+
+from eureka_ml_insights.data_utils import DFTransformBase
+
+
+@dataclass
+class AIMEExtractAnswer(DFTransformBase):
+    model_output_column: str
+    model_books_column: str
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        df[self.model_books_column] = df[self.model_output_column].apply(parse_output_answer)
+        return df
+
+# with all books
+def parse_output_answer(response):
```

Comment:
If this code is taken exactly from somewhere else, please provide a link in the docstring.
Otherwise, if you have written this, please write a test for it.

Reply:
Fixed by adding a unit test aime_utils_tests.py